Remora.Discord.Commands/Feedback/Services/IFeedbackService.cs

```diff
@@ -0,0 +1,128 @@
+//
+//  IFeedbackService.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Threading;
+using System.Threading.Tasks;
+using Remora.Discord.API.Abstractions.Objects;
+using Remora.Discord.Commands.Feedback.Messages;
+using Remora.Rest.Core;
+using Remora.Results;
+
+namespace Remora.Discord.Commands.Feedback.Services;
+
+/// <summary>
+///     Handles sending formatted messages to the users.
+/// </summary>
+public interface IFeedbackService
+{
+    /// <summary>
+    ///     Sends a feedback message with the given content and color in the current context, optionally mentioning a target
+    ///     user.
+    /// </summary>
+    /// <param name="contents">The content of the feedback message.</param>
+    /// <param name="color">The color of the feedback message embed.</param>
+    /// <param name="target">The optional target user to mention in the feedback message.</param>
+    /// <param name="options">Optional custom message options.</param>
+    /// <param name="ct">Optional CancellationToken to cancel the operation.</param>
+    /// <returns>A Result containing a list of sent messages, or an error if the operation failed.</returns>
+    Task<Result<IReadOnlyList<IMessage>>> SendContextualContentAsync(
```

Comment:
Formatting. The opening brace should be placed on a new line if you can't fit it all on the same line, same way you'd handle { and }.

Check the other methods for this as well.